.travis.yml

```diff
@@ -1,27 +1,25 @@
+sudo: required
+dist: trusty
 language: ruby
+before_install: gem install -N bundler -v '~> 1.11'
 cache: bundler
-sudo: false
 rvm:
-- 2.0.0
-- 2.1
-- 2.2
-- 2.3.0
-- jruby
+  - 2.1
+  - 2.2
+  - 2.3
+  - jruby-9.1.5.0
```

Comment:
We are testing against 3 versions of MRI but only one of jruby. I think that at least we should also test against 9.1.1.0. You mentioned in the commit comment that it's the oldest that could work well.